language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/MatchedExpressionNodeResolver.java

```diff
@@ -131,11 +131,11 @@ public MatchedExpressionNodeResolver(Node matchedNode) {
 
     @Override
     public Optional<ExpressionNode> transform(ListConstructorExpressionNode listConstructorExpressionNode) {
-        Optional<Node> expressionNode = listConstructorExpressionNode.expressions().stream()
+        Optional<Node> listMemberNode = listConstructorExpressionNode.listMembers().stream()
                 .filter(expression -> this.matchedNode == expression)
                 .findFirst();
-        if (expressionNode.isPresent() && expressionNode.get() instanceof ExpressionNode) {
-            return Optional.of((ExpressionNode) expressionNode.get());
+        if (listMemberNode.isPresent() && listMemberNode.get() instanceof ExpressionNode) {
```

Comment:
`SpreadMemberNode` will not be an instance of `ExpressionNode` will that be an issue here?

Reply:
Created #35507. @malinthar will send a fix separately once this is merged.